src/components/ActionManager/transactionTypes.js

```diff
@@ -0,0 +1,12 @@
+const transactionTypes = {
```

Comment:
should we put this into `cosmos-js`?

Reply:
Yes! That would awesome

Reply:
Wait, no. I'll just use the `cosmos-api` types.

Reply:
hahahha why did you strike this? ^^

Reply:
I had another comment below, but it got outdated. never mind now. 

It would be great to get some kind of type values out of the api.